{
  "dataType": "CVE_RECORD",
  "dataVersion": "5.1",
  "cveMetadata": {
    "cveId": "CVE-2025-0673",
    "assignerOrgId": "ceab7361-8a18-47b1-92ba-4d7d25f6715a",
    "state": "PUBLISHED",
    "assignerShortName": "GitLab",
    "dateReserved": "2025-01-23T14:01:49.872Z",
    "datePublished": "2025-06-12T11:03:28.366Z",
    "dateUpdated": "2025-06-12T11:03:28.366Z"
  },
  "containers": {
    "cna": {
      "title": "Loop with Unreachable Exit Condition ('Infinite Loop') in GitLab",
      "descriptions": [
        {
          "lang": "en",
          "value": "An issue has been discovered in GitLab CE/EE affecting all versions from 17.7 before 17.10.8, 17.11 before 17.11.4, and 18.0 before 18.0.2, allow an attacker to trigger an infinite redirect loop, potentially leading to a denial of service condition."
        }
      ],
      "affected": [
        {
          "vendor": "GitLab",
          "product": "GitLab",
          "repo": "git://git@gitlab.com:gitlab-org/gitlab.git",
          "cpes": [
            "cpe:2.3:a:gitlab:gitlab:*:*:*:*:*:*:*:*"
          ],
          "versions": [
            {
              "version": "17.7",
              "status": "affected",
              "lessThan": "17.10.8",
              "versionType": "semver"
            },
            {
              "version": "17.11",
              "status": "affected",
              "lessThan": "17.11.4",
              "versionType": "semver"
            },
            {
              "version": "18.0",
              "status": "affected",
              "lessThan": "18.0.2",
              "versionType": "semver"
            }
          ],
          "defaultStatus": "unaffected"
        }
      ],
      "problemTypes": [
        {
          "descriptions": [
            {
              "lang": "en",
              "description": "CWE-835: Loop with Unreachable Exit Condition ('Infinite Loop')",
              "cweId": "CWE-835",
              "type": "CWE"
            }
          ]
        }
      ],
      "references": [
        {
          "url": "https://gitlab.com/gitlab-org/gitlab/-/issues/514732",
          "name": "GitLab Issue #514732",
          "tags": [
            "issue-tracking",
            "permissions-required"
          ]
        },
        {
          "url": "https://hackerone.com/reports/2936949",
          "name": "HackerOne Bug Bounty Report #2936949",
          "tags": [
            "technical-description",
            "exploit",
            "permissions-required"
          ]
        }
      ],
      "metrics": [
        {
          "format": "CVSS",
          "scenarios": [
            {
              "lang": "en",
              "value": "GENERAL"
            }
          ],
          "cvssV3_1": {
            "version": "3.1",
            "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
            "attackVector": "NETWORK",
            "attackComplexity": "LOW",
            "privilegesRequired": "NONE",
            "userInteraction": "NONE",
            "scope": "UNCHANGED",
            "confidentialityImpact": "NONE",
            "integrityImpact": "NONE",
            "availabilityImpact": "HIGH",
            "baseScore": 7.5,
            "baseSeverity": "HIGH"
          }
        }
      ],
      "solutions": [
        {
          "lang": "en",
          "value": "Upgrade to versions 17.10.8, 17.11.4, 18.0.2 or above."
        }
      ],
      "credits": [
        {
          "lang": "en",
          "value": "Thanks [sim4n6](https://hackerone.com/sim4n6) for reporting this vulnerability through our HackerOne bug bounty program",
          "type": "finder"
        }
      ],
      "providerMetadata": {
        "orgId": "ceab7361-8a18-47b1-92ba-4d7d25f6715a",
        "shortName": "GitLab",
        "dateUpdated": "2025-06-12T11:03:28.366Z"
      }
    }
  }
}